model/compile/QtiAssetCompiler/QtiItemAssetCompiler.php

```diff
@@ -105,7 +109,7 @@ private function resolve(ItemMediaResolver $resolver, string $assetUrl, string $
         $mediaAsset = $resolver->resolve($assetUrl);
         $mediaSource = $mediaAsset->getMediaSource();
 
-        if ($mediaSource instanceof HttpSource) {
+        if ($mediaSource instanceof HttpSource || $mediaSource instanceof InlineSource) {
```

Comment:
WDYT: we may stop relying on concrete classes – by adding some new specific interface what would unite such behavior?

Reply:
Hmm, good idea, but not a blocker in my opinion :) Do you think we could use the `MediaManagement` for this distinction?
```suggestion
        if (!$mediaSource instanceof oat\tao\model\media\MediaManagement) {
```